### PR TITLE
feat: add dynamic certificate label and icon

### PR DESCRIPTION
**Pull request by flagrede**
## Description

Closes: #2115
Needs: https://github.com/regen-network/regen-sanity/pull/48

- Use credit class label and icon from sanity, if they exist, to display certificate

---

### Author Checklist

_All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues._

I have...

- [ ] provided a link to the relevant issue or specification
- [ ] provided instructions on how to test
- [ ] reviewed "Files changed" and left comments if necessary
- [ ] confirmed all CI checks have passed

### How to test

1. with sanity credit class: https://deploy-preview-2166--regen-marketplace.netlify.app/certificate/WyJyZXRpcmVtZW50cyIsMSwxMDAyMTcwNSwwLDBd
2. without sanity credit class: https://deploy-preview-2166--regen-marketplace.netlify.app/certificate/WyJyZXRpcmVtZW50cyIsMSw5ODAyNDM3LDAsMF0=

### Reviewers Checklist

_All items are required. Please add a note if the item is not applicable and please **add
your handle next to the items reviewed if you only reviewed selected items**._

I have...

- [ ] confirmed all author checklist items have been addressed
- [ ] reviewed code correctness and readability
- [ ] verified React components follow DRY principles
- [ ] reviewed documentation is accurate
- [ ] reviewed tests
- [ ] manually tested (if applicable)

### CODE REVIEW

**Comment by flagrede** [comment]
@erikalogie you can test that the certificate is now dynamic here (I've updated the C01 credit class in sanity):

- with sanity credit class: https://deploy-preview-2166--regen-marketplace.netlify.app/certificate/WyJyZXRpcmVtZW50cyIsMSwxMDAyMTcwNSwwLDBd
- without sanity credit class: https://deploy-preview-2166--regen-marketplace.netlify.app/certificate/WyJyZXRpcmVtZW50cyIsMSw5ODAyNDM3LDAsMF0=

**Comment by erikalogie** [comment]
@flagrede LGTM!

File: web-components/src/components/certificate/index.tsx [hunk 1]
```diff
@@ -3,7 +3,6 @@ import { Box } from '@mui/material';
 import { headerFontFamily, pxToRem } from '../../theme/muiTheme';
 import { LinkComponentType } from '../../types/shared/linkComponentType';
 import { pluralize } from '../../utils/pluralize';
-import CarbonOffsetBadgeIcon from '../icons/CarbonOffsetBadgeIcon';
```
Comment (blushi) [inline web-components/src/components/certificate/index.tsx:6]: probably we can delete CarbonOffsetBadgeIcon now

**Review by blushi — APPROVED** [review]
Looks good, just a small nit

**Pull request merged**